### ISSUE TITLE
gddo links to original package, not vendored package

### ISSUE THREAD

**Issue by normanvashon** [golang/gddo#435]
The documentation for https://godoc.org/github.com/kubernetes/kubernetes/pkg/api/resource#NewQuantityFlagValue links to https://godoc.org/github.com/spf13/pflag#Value. I expect the documentation to link to https://godoc.org/github.com/kubernetes/kubernetes/vendor/github.com/spf13/pflag.

**Comment by josephspurrier** [comment]
Yeah, I think it would make sense if dependencies were vendored, the documentation would be built from them. For instance, I have vendored dependencies and since they are my own local packages, no documentation shows at all because it's looking online for them: https://godoc.org/github.com/josephspurrier/gowebapp

This is not a feature of godoc by default, but I feel like it would be a good addition.

**Comment by shurcooL** [comment]
@josephspurrier Agreed, that should work. This can be summarized as "vendoring" is not supported. It didn't exist when gddo was created, and it's a feature that no one has added support for here since then.

**Comment by shurcooL** [comment]
There is some minimal support for /vendor/ directory in determining the score of a package for ranking/display purposes.

https://github.com/golang/gddo/blob/aa8084e58a66f26ac6f479a75f0da68ed2067d4e/database/index.go#L169
https://github.com/golang/gddo/blob/aa8084e58a66f26ac6f479a75f0da68ed2067d4e/database/index.go#L215-L216

But that's where support currently stops.
